### DISCUSSION TITLE
systemd.networkd error

### DISCUSSION THREAD

**Discussion by prince-taker** [chpc-tech-eval/scc#127] [Q&A] [answered]
Group Name: Ctrl Alt Elite
We get the following error when trying to run the command: sudo systemctl status systemd.networkd

![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/2a0a611c-565b-4290-9950-bf3ab2900d6c)

**Comment by Dc-Spears** [comment, marked as answer]
Hi prince-taker, 

there is a small spelling error with the name of the service it should be systemd-networkd, and you might still need to install the service.

![systemd-networkd](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/be35a3e9-ddd6-43b2-9c69-797d5a5eeb98)
![system-networkd-status](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/a7cda2dd-57f5-4599-813f-8d6ef80ce820)

**Reply by prince-taker** [reply]
Hi Dc-Spears, 

I get the following error when trying to do that. I tried on different Wi-Fi networks, but still.
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/f3d4ac20-2088-4946-9011-476e188a0b8e)

**Reply by Dc-Spears** [reply]
Hi, did you try updating your OS and installing dependencies.
Try this:
sudo dnf check-update
sudo dnf install dnf-utils
sudo dnf install epel-release

**Reply by nyameko** [reply]
Excellent work @Dc-Spears!
